### PR TITLE
Add brendandburns to kubernetes-client github admins.

### PR DESCRIPTION
**Pull request by brendandburns**
@nikhita @justaugustus 

Let's try this again, many thanks!

Ref #618 #615

### CODE REVIEW

**Comment by brendandburns** [comment]
@justaugustus this is now passing, thanks for the assist!

**Comment by justaugustus** [comment]
Happy to help! :)
/lgtm
/meow space
/assign @nikhita

File: config/kubernetes-client/org.yaml [hunk 1]
```diff
@@ -5,6 +5,7 @@ has_organization_projects: true
 has_repository_projects: true
 members_can_create_repositories: false
 admins:
+- brendandburns
```
Comment (nikhita) [inline config/kubernetes-client/org.yaml]: Only the GitHub admin team can be in the `admins` list: https://github.com/kubernetes/community/tree/master/github-management#github-administration-team :grimacing: 

@brendandburns if **org** admin permissions are needed for setting up something, could you open an issue on this repo? The github admin team can handle that. :)

In general,  you should have admin permissions across all repos....except the python ones. Could you leave only the python change in this PR? Thanks!

Comment (brendandburns) [reply config/kubernetes-client/org.yaml]: @nikhita I filed https://github.com/kubernetes/org/issues/635

Thanks.

**Comment by nikhita** [comment]
@brendandburns can you edit to scope this PR to adding yourself as an admin to the python repos?

**Comment by nikhita** [comment]
closing since this was addressed in https://github.com/kubernetes/org/pull/665.

@brendandburns you should now have admin access to all kubernetes-client repos :)

**Pull request closed without merging**